jumpscale/sals/vdc/deployer.py

```diff
@@ -760,7 +760,6 @@ def renew_plan(self, duration):
                 f"renew plan: extending pool {pool_id}, sus: {sus}, cus: {cus}, reservation_id: {pool_info.reservation_id}"
             )
             self.pay(pool_info)
-        self.vdc_instance.expiration = self.vdc_instance.expiration.timestamp() + duration * 60 * 60 * 24
```

Comment:
I don't see in your code that u extend the expiration date with the new period,`60 * 60 * 24`